components/zip.js

```diff
@@ -0,0 +1,39 @@
+// zip.js
```

Comment:
I choose this name as the process is a classic convolution function, maps a tuple of sequences into a sequence of tuples, which is more commonly referred to as zip.